material/frontend/views/list.py

```diff
@@ -233,11 +234,18 @@ def get_headers_data(self):
             attr = self.get_data_attr(field_name)
             yield field_name, attr.label
 
+    def _boolean_column(self, value):
+        if value:
+            return render_to_string(template_name='material/icons/true.html')
```

Comment:
Isn't this too complicated? Could we just return an html string here?

Detail view actually could have material/readonly/fields/boolean_input.html template to show the icon instead of value.